src/components/BookCard/index.js

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import {
+  BookCardWrapper,
+  BookSectionOne,
+  BookCategory,
+  BookContainer,
+  BookTitle,
+  BookAuthor,
+  BookCommandsWrapper,
+  Comments,
+  BookRemove,
+  BookEdit,
+  BookSectionTwo,
+  CircleCenter,
+  CompletedWrapper,
+  CompletedPerc,
+  CompletedText,
+  BookSectionThree,
+  BookChapter,
+  BookLesson,
+  UpdateProgress,
+  BookLineDivider,
+} from './BookCardElements';
+
+const BookCard = () => (
+  <BookContainer>
+    <BookCardWrapper>
+      <BookSectionOne>
+        <BookCategory>Prayer Book</BookCategory>
+        <BookTitle>Dune</BookTitle>
+        <BookAuthor>Michael Oladele</BookAuthor>
+        <BookCommandsWrapper>
+          <Comments>Comments</Comments>
+          <BookRemove>Remove</BookRemove>
+          <BookEdit>Edit</BookEdit>
```

Comment:
- [ ] I think you are repeating the week, I do suggest keeping all the code just related to the specific **PR**, in this part you have to make the book as props and pass them here.

Reply:
in your bookform, you have the codes that can add books to the card. then you can make them as props type and call the function to render the books in the cards.

Reply:
Great! This will be looked into.
But do you think the project should be disapproved because of this?

Reply:
This issue is well resolved! ✔️